docs/cors.md

```diff
@@ -7,8 +7,12 @@ Starting from ORDS `3.0.3`, a new security rule concerning `Cross Origin Sharing
 
 This occurs when the server is using ORDS `3.0.3`, `3.0.4` or `3.0.5`.
 
-For Apache, please follow the message and set the ProxyPreserveHost parameter to On.
-
+For Apache, there are 2 options
```

Comment:
@krisrice Are there any non-Apache alternative options to `ProxyPreserveHost`? I've been using this pre-built VM that doesn't have Apache: http://www.oracle.com/technetwork/database/enterprise-edition/databaseappdev-vm-161299.html.

(This is a follow-up to https://twitter.com/vincentmorneau/status/752512842436313089)